client/eth.go

```diff
@@ -50,15 +47,35 @@ func (c *EthClient) TxPoolContent(ctx context.Context) (*TxPoolContent, error) {
 	return content, err
 }
 
-// TraceTransaction returns a transaction trace
```

Comment:
Why did you kill all the comments?

Reply:
These comments are pretty useless imo, reverted in 5d40801 (will address in another PR)